myapp/getClients.js

```diff
@@ -0,0 +1,17 @@
+const request = require('request');
+const app = require('./app.js');
+
+const getClients = function () { 
+  app.get('/clients', function (req, res) {
+    request('https://api.createsend.com/api/v3.1/clients.json', function (err, response, data) {
+      if(err) {
+        res.send(err);
+        return(err);
```

Comment:
Line 8-9 can be combined into one,
`return res.send(err);`
This will serve 2 purposes
- return breaks the further execution in case of error
- Error is sent in the response.